dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESIndexUtilTest.java

```diff
@@ -0,0 +1,199 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.content.elasticsearch.util.RestHighLevelClientProvider;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.rainerhahnekamp.sneakythrow.Sneaky;
+import org.elasticsearch.ElasticsearchStatusException;
+import org.elasticsearch.action.admin.indices.settings.put.UpdateSettingsRequest;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.BeforeClass;
+import com.dotcms.util.IntegrationTestInitService;
+import org.junit.Test;
+
+import static org.jgroups.util.Util.assertEquals;
+
+public class ESIndexUtilTest {
+
+    @BeforeClass
+    public static void prepare () throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index is not read only
+     * Should:return false
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldReturnFalseIfTheCurrentIndicesAreReadOnly() throws DotDataException {
```

Comment:
I think the test name should be changed because you are testing a _not_ read only scenario

Reply:
exactly

Reply:
done https://github.com/dotCMS/core/pull/18604/commits/fc979e0e1d04475951dbb213b379b17022474a8e#diff-1e54965847e19cc0a77583c4dbd18afdR36